packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -55,67 +61,125 @@ const CompleteResetPassword = ({
 
   return (
     <AppLayout>
-      <CardHeader
-        headingText="Create new password"
-        headingTextFtlId="complete-reset-pw-header"
-      />
+      <p className="text-start text-grey-400 text-sm">
+        <FtlMsg id="password-reset-flow-heading">Reset your password</FtlMsg>
+      </p>
 
-      {!hasConfirmedRecoveryKey &&
-        locationState.recoveryKeyExists === undefined && (
-          <Banner type={BannerType.error}>
-            <>
-              <FtlMsg id="complete-reset-password-recovery-key-error-v2">
-                <p>
-                  Sorry, there was a problem checking if you have an account
-                  recovery key.
-                </p>
-              </FtlMsg>
-              <FtlMsg id="complete-reset-password-recovery-key-link">
+      {/*
+        In the event of serious error. A bright red banner will be displayed indicating
+        the problem.
+      */}
+      {errorMessage && <Banner type={BannerType.error}>{errorMessage}</Banner>}
+
+      {/*
+        Inline message to help the user understand the status and ramifications
+        of their password reset. ie Will their data persist, or is an way to
+        recover it.
+      */}
+      {hasConfirmedRecoveryKey === false &&
```

Comment:
This error should only be shown if there was an error checking if the user has a recovery key (if `recoveryKeyExists` is undefined) - very much an edge case. 

We will be updating the banner styling to change the color and icon consistently throughout the app in https://mozilla-hub.atlassian.net/browse/FXA-9714 - until then this banner could use the regular error banner styling (or we can add a note to that ticket to replace this custom message with the updated banner if we opt to keep this one as is).

Reply:
Thanks for clarifying, that makes more sense. I was a little fuzzy on this based on the ticket alone. I'll add a note to FXA-9714 so as not confuse QA during review, as this banner matches what's in figma.